frontend/src/app/modules/imports/services/data.service.ts

```diff
@@ -17,21 +18,30 @@ import {
   ContentMappingValues,
 } from '../models/mapping.model';
 import { ConfigService } from '@geonature/services/config.service';
+import { ActivatedRoute } from '@angular/router';
 
 @Injectable()
 export class DataService {
   private urlApi = null;
+  private destination: string = null;
 
   constructor(
     private _http: HttpClient,
     public config: ConfigService
   ) {
-    this.urlApi = `${this.config.API_ENDPOINT}/import/synthese`;
-    // this.urlApi = `${this.config.API_ENDPOINT}/import/occhab`;
+    this.urlApi = `${this.config.API_ENDPOINT}/import`;
+  }
+
+  setDestination(destination: string) {
+    this.destination = destination;
+  }
+
+  getUrlApi() {
+    return `${this.urlApi}/${this.destination}`;
   }
 
   getNomenclatures(): Observable<any> {
-    return this._http.get<any>(`${this.urlApi}/nomenclatures`);
+    return this._http.get<any>(`${this.getUrlApi()}/nomenclatures`);
   }
```

Comment:
**Suggestion:** It's a good practice to handle potential errors in HTTP requests. Consider adding error handling for the HTTP requests. [best practice]
```suggestion
  getNomenclatures(): Observable<any> {
      return this._http.get<any>(`${this.getUrlApi()}/nomenclatures`).pipe(
          catchError(this.handleError)
      );
  }

  private handleError(error: HttpErrorResponse) {
      // transform error response to user friendly message
      let errMsg = 'Unknown error occurred';
      if (error.error instanceof ErrorEvent) {
          errMsg = `An error occurred: ${error.error.message}`;
      } else {
          errMsg = `Server returned code: ${error.status}, error message is: ${error.message}`;
      }
      return throwError(errMsg);
  }
```